.github/workflows/build.yml

```diff
@@ -431,19 +431,14 @@ jobs:
           echo $PATH
           
           cd ./cmd/wallet-sdk-js
-          sed -i 's/"version": "0.1.0"/"version": "'${ env.NEW_VERSION }'"/g' package.json
+          sed -i 's/"version": "0.1.0"/"version": "'${{ env.NEW_VERSION }}'"/g' package.json
```

Comment:
Why we needed double brackets here?

Reply:
We need them to use the actual value of the `NEW_VERSION` environment variable that's set in the previous step